### DISCUSSION TITLE
Version 1.7.15 add a hidden tag and what is this do?

### DISCUSSION THREAD

**Discussion by S-YOU** [tailwindlabs/headlessui#2519] [Help]
Version 1.7.15 add a hidden tag and what is this exactly do and why needed?

```html
<div style="position:fixed;top:1px;left:1px;width:1px;height:0;padding:0;margin:-1px;overflow:hidden;clip:rect(0, 0, 0, 0);white-space:nowrap;border-width:0;display:none"></div>
````

**Comment by rtconner** [comment]
Yeah this is annoying me too because it slightly effects SEO/page speed in a negative way. Can I have this tag use tailwind classes instead of the style attribute?
